charts/kafka-operator/values.yaml

```diff
@@ -34,6 +34,9 @@ operator:
 
 webhook:
   enabled: true
+#  serverPort:
+  tls:
+    certDir: ""
```

Comment:
```suggestion
    # certDir: ""
```

Reply:
I get an error of `nil pointer evaluating interface {}.certDir` when I comment either this line or both this line and the`tls` line out